src/server/scripts/OutdoorPvP/OutdoorPvPNA.cpp

```diff
@@ -533,12 +580,34 @@ int32 OPvPCapturePointNA::HandleOpenGo(Player* player, GameObject* go)
 
 bool OPvPCapturePointNA::Update(uint32 diff)
 {
-    // let the controlling faction advance in phase
-    bool capturable = false;
-    if (m_ControllingFaction == TEAM_ALLIANCE && m_activePlayers[0].size() > m_activePlayers[1].size())
-        capturable = true;
-    else if (m_ControllingFaction == TEAM_HORDE && m_activePlayers[0].size() < m_activePlayers[1].size())
-        capturable = true;
+    if (!m_capturePoint)
+        return false;
+
+    float radius = ((float)m_capturePoint->GetGOInfo()->capturePoint.radius) - 60.0f;
```

Comment:
I get the reasoning of the area not being this big. But how does it work on classic?

Reply:
> I get the reasoning of the area not being this big. But how does it work on classic?

I don't know how it works, but i saw a recent video that the area now at TBC Classic it was more big so im going to delete that reduce area capture:

https://youtu.be/xHM_78DFM7c?si=wPirLfGnkbftQXGy&t=25